spec/index.html

```diff
@@ -1428,7 +1428,7 @@ <h5>The <code>&lt;<dfn id="the-mapml-element">mapml</dfn>&gt;</code> element</h5
           <dd>
             <pre class="idl">
             [Exposed=Window]
-            interface MAPMLMapMLElement : MAPMLElement {
+            interface HTMLHtmlElement : HTMLElement {
```

Comment:
I think at the moment this should be "interface HTMLMapmlElement : HTMLElement".  @zcorpan and I did not close the loop on this conversation, but the reasoning is that the <mapml> root document element would still be necessary for a file that is read from a file system and assigned to the browser application by its "filename.ext" file system file type extension mapping, but for which the browser has no previous hint to go by, such as a <!DOCTYPE ...> declaration (In such a case the browser would be expected, as it does with, for example, "myfile.png", synthesize an HTML document enclosing the markup that would render the map layer with default starting conditions, if possible).

Reply:
Okay, changed interface to `HTMLMapmlElement` in https://github.com/Maps4HTML/MapML/pull/142/commits/0de8500ac997ea223c56df94b6ba9b1998cdf107.

Reply:
Another thing though, we're currently describing the `map` element as a media element:

> The [`map`](https://maps4html.org/MapML/spec/#the-map-element) element, in conjunction with child [`layer`](https://maps4html.org/MapML/spec/#the-layer-element) and optionally [`area`](https://maps4html.org/MapML/spec/#the-area-element) elements, defines a map [media element](https://html.spec.whatwg.org/multipage/media.html#media-element).

I believe that means the element interface for `<map>` should be "HTMLMapmlElement : [HTMLMediaElement](https://html.spec.whatwg.org/multipage/media.html#htmlmediaelement)".

But, I am responsible for linking to the "[media element](https://html.spec.whatwg.org/multipage/media.html#media-element)" WHATWG definition, which I first should ask, did you intend to define the map element as a "media element" per that definition?